app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -55,7 +55,7 @@
 // $rem-base: 16px;
 
 // Allows the use of rem-calc() or lower-bound() in your settings
-@import "foundation/functions";
+@import 'foundation/functions';
```

Comment:
Yeah I think so - it looks like they use [this config](https://raw.githubusercontent.com/thoughtbot/hound/master/config/style_guides/scss.yml)